tests/conformance/state/state.go

```diff
@@ -223,10 +229,14 @@ func ConformanceTests(t *testing.T, props map[string]string, statestore state.St
 			for _, scenario := range scenarios {
 				if !scenario.bulkOnly && !scenario.transactionOnly {
 					t.Logf("Setting value for %s", scenario.key)
-					err := statestore.Set(&state.SetRequest{
+					req := &state.SetRequest{
 						Key:   scenario.key,
 						Value: scenario.value,
-					})
+					}
+					if config.ComponentName == componentRedis && len(scenario.contentType) != 0 {
```

Comment:
Please, keep the if clause but remove the condition on component name. Since contentType is optional, it will just be ignored by other state stores. The goal is to keep the conformance tests as independent as possible to component specifics. In this case, passing an optional param on all is OK.

Reply:
We might need to ave certification tests for Redis now so we don't pollute more of the conf tests with component specific scenarios.

Reply:
I will create an issue to update certification tests